src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/http_action/config/PerformanceAnalyzerResourceProvider.java

```diff
@@ -89,16 +88,22 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
         SSLContext sc = SSLContext.getInstance("SSL");
         sc.init(null, trustAllCerts, new java.security.SecureRandom());
         HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
-      } catch (NoSuchAlgorithmException e) {
-        LOG.error("Error encountered while initializing SSLContext", e);
-      } catch (KeyManagementException e) {
-        LOG.error("Error encountered while initializing SSLContext", e);
+      } catch (AccessControlException e) {
+        LOG.info("SecurityManager forbids setting default SSL Socket Factory...using default settings", e);
+      } catch (Exception e) {
+        LOG.info("Error encountered while initializing SSLContext...using default settings", e);
```

Comment:
Any specific reason we are using log level as INFO instead of ERROR here?

Reply:
Changed to WARN